uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/interaction/microsoftsts/AdfsLoginComponentHandler.java

```diff
@@ -54,7 +54,14 @@ public void handleEnrollmentPrompt(@NonNull final String username, @NonNull fina
         // handle AAD login page for username
         UiAutomatorUtils.handleInput("i0116", username);
         UiAutomatorUtils.handleButtonClick("idSIButton9");
+    }
 
+    /**
+     * Enters username, password in email, password fields of a login page
+     */
+    public void handlePrompt(@NonNull final String username, @NonNull final String password) {
```

Comment:
I'm not a fan of this. I mean the prompt handler is supposed to handle prompts...the login component handler is supposed to just handle specific login components

Reply:
Are you suggesting we should have a separate prompt handler for this?